driver/ppm_events_public.h

```diff
@@ -1606,22 +1606,36 @@ enum extra_event_prog_code
 	PPM_SC_X(EPOLL_WAIT_OLD, 379) \
 	PPM_SC_X(RSEQ, 380) \
 	PPM_SC_X(CREATE_MODULE, 381) \
-	PPM_SC_X(PIDFD_GETFD, 382) \
-	PPM_SC_X(SCHED_GETATTR, 383) \
-	PPM_SC_X(FACCESSAT2, 384) \
-	PPM_SC_X(_SYSCTL, 385) \
-	PPM_SC_X(QUERY_MODULE, 386) \
-	PPM_SC_X(GET_MEMPOLICY, 387) \
-	PPM_SC_X(SYNC_FILE_RANGE, 388) \
-	PPM_SC_X(PROCESS_MADVISE, 389) \
-	PPM_SC_X(MEMBARRIER, 390) \
-	PPM_SC_X(IOPL, 391) \
-	PPM_SC_X(CLOSE_RANGE, 392) \
-	PPM_SC_X(FANOTIFY_MARK, 393)
+	PPM_SC_X(SCHED_GETATTR, 382) \
+	PPM_SC_X(FACCESSAT2, 383) \
+	PPM_SC_X(_SYSCTL, 384) \
+	PPM_SC_X(QUERY_MODULE, 385) \
+	PPM_SC_X(GET_MEMPOLICY, 386) \
+	PPM_SC_X(SYNC_FILE_RANGE, 387) \
+	PPM_SC_X(PROCESS_MADVISE, 388) \
+	PPM_SC_X(MEMBARRIER, 389) \
+	PPM_SC_X(IOPL, 390) \
+	PPM_SC_X(CLOSE_RANGE, 391) \
+	PPM_SC_X(FANOTIFY_MARK, 392)
+
+// Tracepoint based events start from 1 << 10 (1024)
```

Comment:
1024 syscalls seems like a lot but we're already halfway through the space:

```
usr/include/asm-generic/unistd.h
855:#define __NR_syscalls 436

arch/microblaze/include/asm/unistd.h
41:#define __NR_syscalls         399

arch/x86/include/generated/uapi/asm/unistd_64.h
366:#define __NR_syscalls 449

arch/x86/include/generated/uapi/asm/unistd_32.h
444:#define __NR_syscalls 449

arch/x86/include/generated/uapi/asm/unistd_x32.h
355:#define __NR_syscalls 548
```

Reply:
Should we go for 2048? Seems like a reasonably-big number.